lib/spire/order.rb

```diff
@@ -149,13 +157,16 @@ def create(options)
         client.create(
           :order,
           'customer' => options[:customer],
+          'status' => options[:status]
```

Comment:
@Dujota I think you're missing a comma here.

Reply:
@caseyli comma added, PR updated